doc/user-guide/user-guide.rst

```diff
@@ -122,107 +126,262 @@ To deal with this process in a more comprehensive way, this repository uses a di
+  If you do not have a local copy of the repository, clone it with:
 
-.. prompt:: bash
+  .. _clone-repo:
 
-  git clone https://github.com/lsst-ts/ts_cycle_build.git
+  .. prompt:: bash
 
-You may need to use the ssh version if you have two factor authentication (which you probably should).
-In this case, do:
+    git clone https://github.com/lsst-ts/ts_cycle_build.git
 
-.. prompt:: bash
+  You may need to use the ssh version if you have two factor authentication (which you probably should).
+  In this case, do:
 
-  git clone git@github.com:lsst-ts/ts_cycle_build.git
+  .. prompt:: bash
 
-To update a local copy do;
+    git clone git@github.com:lsst-ts/ts_cycle_build.git
 
-.. prompt:: bash
+  To update a local existing copy do;
 
-  cd ts_cycle_build/  # Update the path to match the location of the package in your environment
-  git checkout main
-  git pull
+  .. _update-repo:
+  
+  .. prompt:: bash
 
-If you are creating a new cycle, start by creating the cycle build and pushing it to github.
-Considering you are starting cycle 21, do;
+    cd ts_cycle_build/  # Update the path to match the location of the package in your environment
+    git checkout main
+    git pull
 
-.. prompt:: bash
+* Check if a revision is already open or not.
 
-  git checkout cycle/0021
-  git push
+  You can check the repository git history to see if there are any ticket branches already opened or contact people on the `ts-build <https://lsstc.slack.com/archives/CM6NYMG3F>`_ slack channel and ask about any opened revisions.
 
-Then, create a ticket branch to work on:
+  .. important::
 
-.. prompt:: bash
+    We want to avoid, as much as possible, to have more than one revision open at any time.
+    Therefore, it is paramount that you coordinate any work done with the build team.
 
-  git checkout tickets/DM-12345  # Update the ticket number to match the ticket you are working on
+  * If a revision exists, checkout the revision branch:
 
+    .. prompt:: bash
 
-Update the cycle/revision and software versions as needed.
+      git fetch --all
+      git checkout tickets/DM-12345  # existing revision branch
 
-.. important::
+  * If not, create a new revision branch:
 
-  If any of the :ref:`core packages <Core-packages>` version is updated, the cycle number must be incremented, otherwise just increment the revision number and update the software versions.
+    .. prompt:: bash
 
-Both cycle, revision and all remaining software versions are managed by an environment file located in:
+      git checkout -b tickets/DM-12345  # create new branch to work on
 
-.. prompt:: bash
 
-  cycle/cycle.env
+  * Open the ``cycle/cycle.env`` file and update the revision number on the top.
+
+    .. important::
+
+      For now we do not have any protection against reusing an existing revision so make sure you update this number before anything else.
+
+      Eventually this will be enforced by the build system.
```

Comment:
This section seems to suggest that a new revision ALWAYS needs to be made but in reality this only is the case when starting a new revision. Please rephrase.